<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConceptMapConceptEditDialog</class>
 <widget class="QDialog" name="QtConceptMapConceptEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Concept/Relatie bewerken</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_concept">
     <property name="text">
      <string>Concept/Relatie</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_concept"/>
   </item>
   <item>
    <widget class="QLabel" name="label_examples">
     <property name="text">
      <string>Voorbeelden/Toelichting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_examples">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edit_text"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>&amp;Voeg voorbeeld toe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_ok">
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_concept</tabstop>
  <tabstop>list_examples</tabstop>
  <tabstop>edit_text</tabstop>
  <tabstop>button_add</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
